tests/python/unittest/test_ndarray.py

```diff
@@ -346,6 +346,50 @@ def test_ndarray_equal():
     z = 0 == x
     assert (z.asnumpy() == np.ones((2, 3))).all()
 
+def test_ndarray_not_equal():
```

Comment:
I will keep these around for testing ndarray and add new equivalent ones in testOperator for testing symbol.py